content/docs/command-reference/exp/apply.md

```diff
@@ -14,22 +14,25 @@ positional arguments:
 
 ## Description
 
-Restores an `experiment` into the workspace as long as no more Git commits have
-been made after the target experiment (`HEAD` hasn't moved). The `experiment`
-can be referenced by name or hash (see `dvc exp run` for details). This changes
-any files (code, data, <abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.)
-needed to reflect the experiment conditions and results in the workspace.
+Restores an `experiment` into the workspace, as long as we're on the same
+project baseline (Git `HEAD`) as when the target experiment was run. The
+experiment can be referenced by name or hash (see `dvc exp run` for details).
 
-⚠️ Conflicting changes in the workspace are overwritten unless `--no-force` is
-used.
+Specifically, `dvc exp apply` checks out any files or directories needed to
+reflect the experiment conditions and results. This can include both with DVC
+and Git: code, data, <abbr>parameters</abbr>, <abbr>metrics</abbr>, etc.
+
+⚠️ This command will destroy any existing changes in the workspace (Git working
+tree) unless the `--no-force` flag is used.
 
 This is typically used after choosing a target `experiment` with `dvc exp show`
-or `dvc exp diff`, and before committing it to Git (making it
-[persistent](/doc/user-guide/experiment-management#persistent-experiments)).
+or `dvc exp diff`, and before committing it to Git (making it [persistent].
+
+> Note that if a history of [checkpoints] is found in the `experiment`, it will
+> **not** be preserved when applying and committing it.
```

Comment:
Note that `exp branch` can be used to preserve checkpoint history?

Reply:
Good idea, mentioned/ linked to `exp branch` from here.